internal/orchestration/controller.go

```diff
@@ -430,10 +317,18 @@ func (e *Experiment) quit() error {
 	return nil
 }
 
-func getIDs(host string, m map[string][]hotstuff.ID) []uint32 {
+func replicaIDsToU32(host string, replicaMap config.ReplicaMap) []uint32 {
```

Comment:
This can be replaced with this one (in `config/config.go`):
```go
func (r ReplicaMap) ReplicaIDs(host string) []uint32 {
	ids := make([]uint32, 0, len(r[host]))
	for _, opts := range r[host] {
		ids = append(ids, opts.ID)
	}
	return ids
}
```
And similar for the `ClientMap`.